sources/packages/api/src-sql/Types/Rollback-update-coe-status.sql

```diff
@@ -0,0 +1,6 @@
+-- Add Not Required to rollback.
+ALTER TYPE sims.coe_status
+ADD
+    VALUE 'Not Required'
```

Comment:
just wanted to know the business logic, coe_status will be always `required` for an application right?. do you know the `'Not Required'` scenario